benchmark/util/is-native-error.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const common = require('../common');
+
+const args = {
+  true: new Error('test'),
+  falsePrimitive: 42,
+  falseObject: { foo: 'bar' },
+};
+
+const bench = common.createBenchmark(
+  main,
+  {
+    argument: ['true'],
+    version: ['native'],
+    n: [1e6],
+  },
+  {
+    flags: ['--expose-internals', '--no-warnings'],
+  },
+);
+
+function main({ argument, version, n }) {
+  const util = common.binding('util');
+  const types = require('internal/util/types');
+
+  const func = { native: util, js: types }[version].isNativeError;
```

Comment:
Isn't it recently deprecated? https://github.com/nodejs/node/blob/20bf3287c2bdce55077da91b7c405aecdc31fef8/doc/api/deprecations.md?plain=1#L4283

Reply:
Yes, you’re right. util.types.isNativeError() is now deprecated. It’s better to use a direct `[instanceof Error]` check instead. Thanks for the heads up

Reply:
it is deprecated but still supported - we should still not regress on stable releases and this change ensures that @RafaelGSS

Reply:
But, what's the point of this benchmark? I mean, if we see util/isNativeError is slower, what it means afterall? Which API will be directly impacted besides the deprecated API?

Reply:
This comparison ensures that existing users will not experience performance drops or unexpected slowdowns until the API is completely removed. It also aims to provide a reference for future changes or alternatives.